src/main/java/core/mate/academy/model/Excavator.java

```diff
@@ -5,11 +5,27 @@
  * Do not remove no-args constructor
  */
 public class Excavator extends Machine {
```

Comment:
Don't begin class or method implementation with empty lines. The empty lines at the start of the class definition should be removed.